numpy/ctypeslib.py

```diff
@@ -527,8 +568,52 @@ def as_array(obj, shape=None):
 
 
     def as_ctypes(obj):
-        """Create and return a ctypes object from a numpy array.  Actually
-        anything that exposes the __array_interface__ is accepted."""
+        """
+            as_ctypes(obj)
+
+            Create and return a ctypes object from a numpy array.  Actually
+            anything that exposes the __array_interface__ is accepted.
+            Basically, converts a dtype into a ctypes type.
+
+            Parameters
+            ----------
+            dtype : dtype
+                The dtype to convert
+
+            Returns
+            -------
+            ctype
+                A ctype scalar, union, array, or struct
+
+            Raises
+            ------
+            NotImplementedError
+                If the conversion is not possible
+
+            Notes
+            -----
+            This function does not losslessly round-trip in either direction.
+            ``np.dtype(as_ctypes_type(dt))`` will:
+              - insert padding fields
```

Comment:
This note seems fine, I don't think it should have the initial space to render correctly, but it probably does need a blank line after (and maybe before) the bullet points start
```suggestion

             - insert padding fields
```